custom/icds_reports/reports/cas_reach_data.py

```diff
@@ -91,62 +104,83 @@ def get_data_for_daily_usage(date, filters):
-                },
-            ],
-            [
-
-                {
-                    'label': _('Districts covered'),
-                    'help_text': _('Total Districts that have launched ICDS CAS'),
-                    'percent': None,
-                    'value': get_value(awc_this_month_data, 'districts'),
-                    'all': None,
-                    'format': 'number',
-                    'frequency': 'month',
-                },
-                {
-                    'label': _('States/UTs covered'),
-                    'help_text': _('Total States that have launched ICDS CAS'),
-                    'percent': None,
-                    'value': get_value(awc_this_month_data, 'states'),
-                    'all': None,
-                    'format': 'number',
-                    'frequency': 'month',
-                }
-            ]
+    cas_reach_records = [
+        [
+            {
+                'label': _('AWCs Launched'),
+                'help_text': awcs_launched_help_text(),
+                'color': None,
+                'percent': None,
+                'value': awcs,
+                'all': None,
+                'format': 'number',
+                'frequency': 'month',
+                'redirect': 'icds_cas_reach/awcs_covered'
+            },
+            number_of_awc_open_yesterday
+        ],
+        [
+            {
+                'label': _('Sectors covered'),
+                'help_text': _('Total Sectors that have launched ICDS CAS'),
+                'percent': None,
+                'value': get_value(awc_this_month_data, 'supervisors'),
+                'all': None,
+                'format': 'number',
+                'frequency': 'month',
+            },
+            {
+                'label': _('Blocks covered'),
+                'help_text': _('Total Blocks that have launched ICDS CAS'),
+                'percent': None,
+                'value': get_value(awc_this_month_data, 'blocks'),
+                'all': None,
+                'format': 'number',
+                'frequency': 'month',
+            },
+        ],
+        [
+
+            {
+                'label': _('Districts covered'),
+                'help_text': _('Total Districts that have launched ICDS CAS'),
+                'percent': None,
+                'value': get_value(awc_this_month_data, 'districts'),
+                'all': None,
+                'format': 'number',
+                'frequency': 'month',
+            },
+            {
+                'label': _('States/UTs covered'),
+                'help_text': _('Total States that have launched ICDS CAS'),
+                'percent': None,
+                'value': get_value(awc_this_month_data, 'states'),
+                'all': None,
+                'format': 'number',
+                'frequency': 'month',
+            }
         ]
+    ]
+
+    display_lss_launched = current_month > datetime(2020, 1, 1) and show_prerelease_features
```

Comment:
is it starting from Feb? just reconfirming

Reply:
spec says its from Jan

Reply:
Sorry, dumb mistake. Fixed it.